command/format.go

```diff
@@ -347,14 +348,16 @@ func OutputSealStatus(ui cli.Ui, client *api.Client, status *api.SealStatusRespo
 	if err != nil && strings.Contains(err.Error(), "Vault is sealed") {
 		leaderStatus = &api.LeaderResponse{HAEnabled: true}
 		err = nil
-	}
+	} 
 	if err != nil {
 		ui.Error(fmt.Sprintf("Error checking leader status: %s", err))
 		return 1
 	}
+	if err == nil {
```

Comment:
We already return in the previous `if` when err is non-nil, so this change is again gratuitous.

Reply:
I'm blind and keep failing to register that... sorry its some personal bias and habit of mine with explicit conditioning even if too verbose - will revert back to original which was correct given the aforementioned `return` above. Thx again @ncabatoff

Reply:
No worries, we all have our style preferences and it's hard to relax them.